projects/court-case-and-delius/deploy/values-dev.yml

```diff
@@ -15,6 +15,7 @@ generic-service:
     SPRING_DATASOURCE_HIKARI_MINIMUMIDLE: 0
 
     LOGGING_LEVEL_UK_GOV_DIGITAL_JUSTICE_HMPPS: DEBUG
+    COMMUNITY-API_URL: https://community-api-secure.test.delius.probation.hmpps.dsd.io
```

Comment:
Minor thing - but we're better off using these URLs for Community API:
* https://community-api.test.probation.service.justice.gov.uk
* https://community-api.pre-prod.delius.probation.hmpps.dsd.io
* https://community-api.probation.service.justice.gov.uk

They point to the same service so it doesn't matter too much, but people were meant to migrate to the new URLs a while ago.

Reply:
made the changes